utils/statistics-flow-certificates-filter-result.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
```

Comment:
All changes affecting `1764-stats-table-certificates-view` should be made on that branch and not here.

Otherwise when testing, #1764 we will not test final version, and it's not possible to reliably test and accept things gradually